Frends.AzureEventHub.Receive/Frends.AzureEventHub.Receive/Definitions/Checkpoint.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureEventHub.Receive.Definitions;
+
+/// <summary>
+/// Checkpoint parameters.
+/// </summary>
+public class Checkpoint
+{
+    /// <summary>
+    /// Authentication method.
+    /// </summary>
+    /// <example>AuthenticationMethod.ConnectionString</example>
+    [DefaultValue(AuthenticationMethod.ConnectionString)]
+    public AuthenticationMethod AuthenticationMethod { get; set; }
+
+    /// <summary>
+    /// The name of the blob container.
+    /// </summary>
+    /// <example>examplecontainer</example>
+    public string ContainerName { get; set; }
+
+    /// <summary>
+    /// If true, operation creates a new container under the specified account. 
+    /// If the container with the same name already exists, it is not changed.
+    /// </summary>
+    /// <example>false</example>
+    [DefaultValue(false)]
+    public bool CreateContainer { get; set; }
+
+    /// <summary>
+    /// A connection string.
+    /// </summary>
+    /// <example>DefaultEndpointsProtocol=https;AccountName=accountname;AccountKey=Pdlrxyz==;EndpointSuffix=core.windows.net</example>
+    [UIHint(nameof(AuthenticationMethod), "", AuthenticationMethod.ConnectionString)]
+    [PasswordPropertyText]
```

Comment:
Please add annotation for [DisplayFormat(DataFormatString = "Text")] otherwise the input field can default to Expression.